html/elements/link.json

```diff
@@ -0,0 +1,1283 @@
+                  "chrome_android": {
+                    "version_added": "58.0"
+                  },
+                  "edge": {
+                    "version_added": false
+                  },
+                  "edge_mobile": {
+                    "version_added": false
+                  },
+                  "firefox": {
+                    "version_added": "50.0"
+                  },
+                  "firefox_android": {
+                    "version_added": "50.0"
+                  },
+                  "ie": {
+                    "version_added": null
+                  },
+                  "ie_mobile": {
+                    "version_added": null
+                  },
+                  "opera": {
+                    "version_added": null
+                  },
+                  "opera_android": {
+                    "version_added": null
+                  },
+                  "safari": {
+                    "version_added": null
+                  },
+                  "safari_ios": {
+                    "version_added": null
+                  }
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            }
+          },
+          "rel": {
+            "__compat": {
+              "basic_support": {
+                "support": {
+                  "webview_android": {
+                    "version_added": true
+                  },
+                  "chrome": {
+                    "version_added": "1.0"
+                  },
+                  "chrome_android": {
+                    "version_added": true
+                  },
+                  "edge": {
+                    "version_added": true
+                  },
+                  "edge_mobile": {
+                    "version_added": true
+                  },
+                  "firefox": {
+                    "version_added": "1.0"
+                  },
+                  "firefox_android": {
+                    "version_added": "1.0"
+                  },
+                  "ie": {
+                    "version_added": true
+                  },
+                  "ie_mobile": {
+                    "version_added": true
+                  },
+                  "opera": {
+                    "version_added": true
+                  },
+                  "opera_android": {
+                    "version_added": true
+                  },
+                  "safari": {
+                    "version_added": true
+                  },
+                  "safari_ios": {
+                    "version_added": true
+                  }
+                },
+                "status": {
+                  "experimental": false,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              },
+              "alternate stylesheet": {
```

Comment:
`"alternate_stylesheets" `as a key and `desc: "alternate stylesheets"` ? Unsure if I like space in feature keys.